Clarifies several Marketplace and Integrations-Extras integrations steps (#10573)
DataDog/documentation: content/en/developers/marketplace.md

The Datadog Development Toolkit command (`ddev`) allows you to create scaffolding when you are first developing your integration, by spinning up a skeleton of all the assets and metadata for your tile.

Ensure that Python 3.8 or higher is installed (https://www.python.org/downloads/).	

To avoid a range of potential environment conflicts, we recommend going into the directory where you cloned the repository and creating a simple virtual environment with the following command: `python3 -m pip install virtualenv --user`

The latest released version of the Datadog Development Toolkit may then be installed from [PyPI][23]:

`python -m pip install --upgrade "datadog-checks-dev[cli]"`

**Note:** if you are using Z shell, you may need to use escape characters: `python -m pip install --upgrade datadog-checks-dev\[cli\]` 

> Set the location of to the cloned repository:

Author: I removed the path example language, so I think we can keep the original organization.

> #### Marketplace:

Author: I'd like to keep the headings for Marketplace and Integrations-Extras, to be very clear on the difference between the two. We've seen developers get confused on this point, since we are asking them to submit their integration to one or the other.

> `ddev config set marketplace /path/to/marketplace_directory` (For example: /Users/yourname/Documents/marketplace)

Author: Hi @kayayarai, the path I gave is just an example. The partner may have a very different path, so I think it would be confusing to tell them use /Users/yourname/Documents/marketplace directly in the command. 

If we shouldn't include an example, or it's too verbose, I will make the small change so the path reads "path/to/marketpace_directory.

> `ddev config set marketplace /path/to/marketplace_directory` (For example: /Users/yourname/Documents/marketplace)

Author: Also, I believe the single ticks were recommended by the Docs team in the original PR, and they seem to be displaying correctly on the live page: https://docs.datadoghq.com/developers/marketplace/. Are they necessary?

> `ddev config set marketplace /path/to/marketplace_directory` (For example: /Users/yourname/Documents/marketplace)

Author: Added the backticks and removed the example language.

`ddev config set repo marketplace`

> #### Integrations-Extras:

Author: I'd like to keep the original headings, so I remove the example language.

> `ddev config set extras /path/to/integrations-extras_directory` (For example: /Users/yourname/Documents/integrations-extras)

Author: Added the backticks and removed the example language.

`ddev config set repo extras`

#### Tile Only Listing

For standalone software and services--or if your integration is using the Datadog API and does not contain any Python code--the Development Toolkit supports a tile-only command. 

> `cd` into the marketplace or integrations-extras directory and run: 

Reviewer: ```suggestion
In the `marketplace` or `integrations-extras` directory you specified above, run: 
```

> `ddev create -t tile "<Offering Name>"`

Reviewer: Use three-backticks Markdown code-fence for the full-line monospace, not singles

Author: Added the three backticks throughout the doc. Hopefully, I didn't miss any :)

#### Full Integration

> For the complete integration scaffolding, `cd` into the the marketplace or integrations-extras directory and run: 

Reviewer: ```suggestion
To generate a complete integration scaffolding, from the `marketplace` or `integrations-extras` directory specified above, run: 
```

> `ddev create "<Offering Name>"`

Reviewer: Use three-backticks

Author: Done

#### images

* Store all images used in your `README.md` file in the `images` folder. 

> * **Note:** We recommend not using empty spaces in the name of image files.

Reviewer: ```suggestion
* **Note:** Don't include spaces in the name of image files.
```

* You can create dashboards and monitors directly in your sandbox account and export them as JSON files. 
* See our [Dashboarding Best Practices][26] document for details.

#### Logos

* Consists of at least one SVG, which our DesignOps team implements throughout the Datadog app for both light and dark modes. Logo SVG files can be added to the `assets` directory, or you can place them in a `logos` sub-directory under `assets`.
* **Note:** Technology Partners are responsible for the licensing of the logos they submit.  

#### Changelog.md

* Captures release notes and version information, and displays this in the “Release Notes” tab of your tile. Add release notes in descending order (latest version at the top).

#### Codeowners

> * Lives under the shared .github directory and defines the individuals or teams that are responsible for code in the repository. See https://help.github.com/articles/about-codeowners/ for help with syntax.

Reviewer: ```suggestion
* Located in the shared `.github` directory and defines the individuals or teams that are responsible for code in the repository. See [Github's documentation][29] for help with syntax.
```

And add this at the very end of the file:
`[29]: https://help.github.com/articles/about-codeowners/`

Author: Thanks for updating the link reference. 

Could we use "Lives in" instead of "Located in"? I was trying to keep some consistency with the verb tense for each of these sections (e.g., "Consists," "Captures," "Lives"). Also, this allows "Lives" to share the same tense as "defines" later in the line. I'll accept and then add a new commit for now.
